__tests__/lib/components/PrimaryButton-test.js

```diff
@@ -0,0 +1,51 @@
+jest.unmock('../../../lib/components/PrimaryButton.jsx');
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import TestUtils from 'react-addons-test-utils';
+import {shallow} from 'enzyme';
+
+import PrimaryButton from '../../../lib/components/PrimaryButton.jsx';
+
+describe('PrimaryButton', () => {
+
+  let buttonItem = null;
+
+  const text = 'Test Label';
+
+  beforeEach(function() {
+    buttonItem = shallow(<PrimaryButton text={text} />);
+  });
+
+  it('is defined', () => {
+    expect(buttonItem).toBeDefined();
+  });
+
+  it('is a button element', () => {
+    expect(buttonItem.type()).toEqual('button');
+  });
+
+  it('is has specified text', () => {
+    expect(buttonItem.text()).toEqual(text);
+  });
+
+  it('is has a active class', () => {
+    buttonItem = shallow(<PrimaryButton text={text} status={PrimaryButton.STATUS_ACTIVE} />);
+    expect(buttonItem.hasClass('usa-button-active'));
+  });
+
+  it('is has a hover class', () => {
+    buttonItem = shallow(<PrimaryButton text={text} status={PrimaryButton.STATUS_HOVER} />);
+    expect(buttonItem.hasClass('usa-button-hover'));
+  });
+
+  it('is has a disabled status', () => {
+    buttonItem = shallow(<PrimaryButton text={text} status={PrimaryButton.STATUS_DISABLED} />);
```

Comment:
I would think it would be easier and more intuitive to have a `disabled` prop on the button that you can set to make it disabled.  For example `<PrimaryButton text={text} disabled />`.  Thoughts?